### ISSUE TITLE
A11y: DS4 primary button color contrast when focused

### ISSUE THREAD

**Issue by msendlakowski** [eBay/skin#421]
When the DS4 primary button is in focus or hover state the color contrast doesn't meet accessibility standards.  

FYI, this is a p2 bug that needs to be resolved in 8 days.

![bug3501894_select vehicle](https://user-images.githubusercontent.com/105656/47172402-99c64480-d2c8-11e8-9409-ac14616f7dc7.png)

**Comment by msendlakowski** [comment]
You can reproduce by going to the primary button at:
https://ebay.github.io/skin/#button

And hovering over the primary button. (or tabbing into it so it has focus)

The contrast ratio of the text should be larger than 4.5:1 for small text of 3:1 for large text (14 pt bold or 18pt)

This is small text (14px = 10.5pt). When the button is focused or hovered, the foreground is #ffffff over #5187cf, which results in a contrast ratio of 3.67:1.

**Comment by gbeckpdx** [comment]
It's odd, DS4 vs DS6 color are basically swapped as far as our default/hover states go. Can you try **#3665F3 for on-hover?** It's our primary blue in DS6, but probably comes close and should pass. I'll double check with my team.

**Comment by msendlakowski** [comment]
@gbeckpdx One problem is that the color itself is fine, but in skin we change the opacity on hover, so it just appears lighter.

**Comment by ianmcburnie** [comment]
@msendlakowski No complaints from me.

**Comment by j05eph** [comment]
I think I'd prefer buttons to follow the text links pattern - and using a darker color on roll. 
![image](https://user-images.githubusercontent.com/4842487/47232975-1cf9a000-d386-11e8-9e65-59d60e44b5bd.png)

**Comment by msendlakowski** [comment]
@j05eph This is for existing DS4 buttons only.  Do we want to make that drastic a change for the old design system?  Also, this change will be very slow to roll out since its only being made in the latest Skin version.  That means we will have a lot of inconsistency across the site for quite a while.

**Comment by j05eph** [comment]
AH. I think your fix is the way to go then.

**Comment by ianmcburnie** [comment]
Fix will go out in 6.1.0 release.
